cirq-core/cirq/transformers/gauge_compiling/sqrt_iswap_gauge.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A Gauge transformer for SQRT_ISWAP gate."""
+
+from cirq.transformers.gauge_compiling.gauge_compiling import (
+    ConstantGauge,
+    GaugeTransformer,
+    GaugeSelector,
+)
+from cirq import ops
+
+
+SqrtISWAPGaugeSelector = GaugeSelector(
+    gauges=[
+        ConstantGauge(
+            pre_q0=ops.Z, pre_q1=ops.Z, post_q0=ops.Z, post_q1=ops.Z, two_qubit_gate=ops.SQRT_ISWAP
+        ),
+        ConstantGauge(
+            pre_q0=ops.X, pre_q1=ops.X, post_q0=ops.X, post_q1=ops.X, two_qubit_gate=ops.SQRT_ISWAP
+        ),
+        ConstantGauge(
+            pre_q0=ops.Y, pre_q1=ops.Y, post_q0=ops.Y, post_q1=ops.Y, two_qubit_gate=ops.SQRT_ISWAP
+        ),
+    ]
+)
```

Comment:
Should we also have an identity gauge here?

Reply:
added